### PR TITLE
API Docs Update with Providers

### PR DESCRIPTION
**Pull request by leviatan89**

### CODE REVIEW

File: source/includes/_playlist.md [hunk 1]
```diff
@@ -146,6 +152,10 @@ offset (optional) | integer | starting from
 Returns — Matrix of Playlist Models - all playlists belonging to a user grouped by types
 </aside>
 
+<aside class="notice">
+The playlists of type "nexRex" (method playlist.FetchNexRexPlaylists) and "algorithm" (method playlist.FetchBifrostPlaylists) can take few seconds to be generated after the user profile is updated
```
Comment (tricaalexandru) [inline source/includes/_playlist.md:156]: @leviatan89 "can take a few"

File: source/includes/_playlist.md [hunk 2]
```diff
@@ -319,8 +333,10 @@ curl --data-binary '{"id":"5", "method":"playlist.FetchCompetencyPlaylists", "js
 				"restrictionLabel": "Free",
 				"provider": {
 					"id": 56,
-					"name": "",
-					"image": ""
+					"name": "TED",
+					"image": "/shot/p/sq/56.png",
```
Comment (tricaalexandru) [inline source/includes/_playlist.md:337]: @leviatan89 is this enough for them to get the image?

File: source/includes/_playlist.md (hunk 1 above)
Comment (leviatan89) [reply source/includes/_playlist.md]: updated!

File: source/includes/_playlist.md (hunk 2 above)
Comment (leviatan89) [reply source/includes/_playlist.md]: added all the url

**Pull request merged**